i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

```diff
@@ -361,6 +361,46 @@ By:
 
 ## Centreon Collect
 
+### 22.04.2
+
+Release date: `February 1, 2023`
+
+#### Centreon Engine
+
+##### Improvements
+
+- Improved network connection stability by sending tcpkeepalive packet every 30s on tcp stream.
+
+##### Bug fixes
+
+- Fixed a bug that made deleted acknowledgements come back after restarting Centengine.
+- Fixed log flushing issues in Engine and Broker by improving the reliability and configurability of the flush period.
+
+#### Centreon Broker
+
+##### Improvements
+
+- Improved Broker's behaviour when trying to insert negative values in unsigned columns of the storage database.
+- Improved network connection stability by sending tcpkeepalive packet every 30s on tcp stream.
+- Improved the manner of sending bulk data with Broker on MariaDB databases.
+- It is now possible to use empty strings for parameter values in stream connectors.
+
+##### Bug fixes
+
+- Fixed an issue causing pollers to be displayed as running, and resources to be displayed in Resources Status after stopping centengine.
+- Fixed a bug where deleted host groups and service groups weren’t available in the filter menu.
+- Fixed a deadlock issue due to new Broker stats engine.
+- Fixed a deadlock issue when a query fails while inserting a value out of range for the column type.
+- Fixed an issue due to a data type mismatch that could block Broker.
+- Fixed the way Broker handled numbers with an out of range value for the column type.
+- Fixed a bug that made deleted acknowledgements come back after restarting Centengine.
+- Fixed an issue in RRD rebuild that recreated the RRD files for every 24 hours of data, resulting in keeping only the last day's data.
+- Fixed the issue with RRD rebuild compatibility with rrdcached.
+- Fixed a stability issue that could cause Broker to crash when a host was deleted.
+- Fixed log flushing issues in Engine and Broker by improving the reliability and configurability of the flush period.
+- Fixed an issue where Broker displayed the expected warning message “Deprecated endpoint found in the output configuration: 'file' endpoint is deprecated and should not be used anymore” instead of an error.
```

Comment:
```suggestion
- Fixed an issue where Broker displayed the warning message “Deprecated endpoint found in the output configuration: 'file' endpoint is deprecated and should not be used anymore” instead of an error.
```